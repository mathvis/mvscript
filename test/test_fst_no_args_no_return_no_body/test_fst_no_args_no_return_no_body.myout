(Decl (FunctionDef (FunctionIdentifier "a") [] VoidT (Just (Block (FunctionBlock VoidT) []))),ParserState {config = Configuration {debug = True, collapseOperations = True, collapseControlFlow = True, colors = True}, st = fromList [], fst = fromList [("a",FunctionData {returnType = VoidT, arguments = [], hasBody = True})], context = Nothing, unresolvedFunctionCalls = []})
