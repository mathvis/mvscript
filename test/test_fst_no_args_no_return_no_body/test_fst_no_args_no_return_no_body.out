Decl (FunctionDef (FunctionIdentifier "a") [] void (Just (Block (FunctionBlock void) [])))

--- STATE ---

Function symbol table:
	Function name: a
		Return type: void
		Arguments: 
		Has body: True
